### PR TITLE
[RHEL/6] [RHEL/7] Improve the formatting for setting the login banner

### PR DESCRIPTION
**Pull request by ykhodorkovskiy**
The login banner will now fit to an 80 character screen and have new
lines between bullet points.

### CODE REVIEW

**Comment by iankko** [comment]
Thank you for the proposal. Though the system banner looks better now, it's not sufficient to update the remediation fixes (*/input/fixes file).

The current form returns ```error``` when attempting remediation (meaning that the ```oscap``` command line tool attempted the remediation fix - which sets the login banner to rows length up to 80 characters at maximum, but then when verifying if the banner is set to the expected form the check failed).

Long story short, it's necessary to update also the ```login_banner_text``` variable definition in ```*/input/system/account/banners.xml``` for all products using this variable to match the new (max 80 characters at row) form.

NACK for now (till the issue is fixed).

**Comment by ykhodorkovskiy** [comment]
I just pushed another commit that fixes the regexes in the variables. Let me know if you want me to squash the commit into the previous one.

**Comment by iankko** [comment]
Verified this version works fine on all three systems - RHEL-6, RHEL-7 & Fedora 20.

(On Fedora needed to supply existing RHEL-7 remediation fix ```set_system_login_banner.sh``` the remediation to be actually performed, but once supplied it works fine on Fedora 20 too).

Thank you for the correction. ACK & merging.
